third_party/move/tools/move-unit-test/Cargo.toml

```diff
@@ -15,34 +15,30 @@ better_any = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 codespan-reporting = { workspace = true }
 colored = { workspace = true }
+evm = { workspace = true, optional = true }
 evm-exec-utils = { path = "../../evm/exec-utils", optional = true }
 itertools = { workspace = true }
-move-binary-format = { path = "../../move-binary-format" }
-move-bytecode-utils = { path = "../move-bytecode-utils" }
-once_cell = { workspace = true }
-rayon = { workspace = true }
-regex = { workspace = true }
-
-move-command-line-common = { path = "../../move-command-line-common" }
-move-compiler = { path = "../../move-compiler" }
-move-compiler-v2 = { path = "../../move-compiler-v2" }
-move-core-types = { path = "../../move-core/types" }
-move-ir-types = { path = "../../move-ir/types" }
-move-model = { path = "../../move-model" }
-move-package = { path = "../move-package" }
-move-resource-viewer = { path = "../move-resource-viewer" }
+move-binary-format = { workspace = true }
+move-bytecode-utils = { workspace = true }
+move-command-line-common = { workspace = true }
+move-compiler = { workspace = true }
+move-compiler-v2 = { workspace = true }
+move-core-types = { workspace = true }
+move-ir-types = { workspace = true }
+move-model = { workspace = true }
+move-package = { workspace = true }
+move-resource-viewer = { workspace = true }
 move-stdlib = { path = "../../move-stdlib", features = ["testing"] }
-move-symbol-pool = { path = "../../move-symbol-pool" }
+move-symbol-pool = { workspace = true }
 move-table-extension = { path = "../../extensions/move-table-extension" }
```

Comment:
Could be made `{ workspace = true }`?

Reply:
Here it is the same as for `move-stdlib`. We have a copy of table extension in our aptos tree due to gas charging (`aptos-move-table-extension` or `aptos-table-extension` iirc), and so this one is actually not used, so we should not add it to the workspace to avoid having two extensions :)